app/helpers/application_helper.rb

```diff
@@ -1,2 +1,28 @@
 module ApplicationHelper
+  def sortable(column, username = nil)
+    username ||= column.titleize
+    direction = (column == params[:sort] && params[:direction] == "asc") ? "desc" : "asc"
+    link_to username, sort: column, direction: direction
+  end
+
+  def countries_list
+    CS.countries.map { |c| [c[1], c[0]] }
+  end
+
+  def final_price
```

Comment:
```suggestion
  def final_price(cart)
```
remove instance variable